### DISCUSSION TITLE
3d Visualization

### DISCUSSION THREAD

**Discussion by DavideWasTaken** [dvdoug/BoxPacker#585] [Q&A] [answered]
Hi! Thanks for the nice work!

I wonder if it is possible to try and modify the 3dviewer even in the current version or do we have to wait for 4.0.

Thank you!

**Comment by dvdoug** [comment]
Hello @DavideWasTaken 

>  possible to try and modify the 3dviewer

What did you have in mind?

**Reply by DavideWasTaken** [reply]
Hi, I'm thinking of creating 3d models for various boxes in both size and color. So I was wondering if I could already use a 3dviewer or would it need to be developed.

Thank you!

**Reply by dvdoug** [reply, marked as answer]
Hello

The online viewer that's part of v4 is subject to change at anytime until release (particularly the input format, am still experimenting with that), but there's a standalone prototype viewer that's included with v3 - see https://github.com/dvdoug/BoxPacker/blob/3.x/visualiser/babylonjs.html. You're very welcome to take that as a base and work from it

**Reply by DavideWasTaken** [reply]
Thank you!
